Source/Manager/IdentityAPI/Models/UserDevice.swift

```diff
@@ -0,0 +1,59 @@
+//
+// Copyright 2011 - 2018 Schibsted Products & Technology AS.
+// Licensed under the terms of the MIT license. See LICENSE in the project root.
+//
+
+import Foundation
+
+/**
+ The user device data.
+
+ SeeAlso: https://techdocs.spid.no/types/device-fingerprint/
+ */
+public struct UserDevice: JSONParsable {
+    ///
+    private var hash: String?
+    private var deviceId: String?
+    private var applicationName: String?
+    private var applicationVersion: String?
+    private var platform: String?
+    ///
+    public init(
+        hash: String? = nil,
+        applicationName: String? = nil,
+        applicationVersion: String? = nil
+    ) {
+        self.hash = hash
+        self.deviceId = UIDevice.current.identifierForVendor!.uuidString
```

Comment:
Is this the best identifier available on iOS? [Documentation](https://developer.apple.com/documentation/uikit/uidevice/1620059-identifierforvendor) says: "A different value is returned for apps on the same device that come from different vendors [...]", meaning it's only per app and not per device. Also is there some value we can use if `identifierForVendor` is `nil`, or does that not even happen? 🤔